portable/Common/mpu_wrappers_v2.c

```diff
@@ -485,7 +485,7 @@
 
             lIndex = ( int32_t ) xTask;
 
-            if( IS_EXTERNAL_INDEX_VALID( lIndex ) != pdFALSE )
```

Comment:
Operands of != had different essential types (bool and int)

Reply:
`pdTRUE` and `pdFALSE` are regarded as boolean type by coverity because of the name contain true or false. In previous MISRA fix, `pdTRUE` is replaced with `pdPASS` and `pdFALSE` is replaced with with `pdFAIL` to avoid these false positive warnings. [Reference link](https://github.com/FreeRTOS/FreeRTOS-Kernel/blob/ba1e2dad3cc6fcb4209e33093efba83d189aa836/examples/coverity/CMakeLists.txt#L8). Suggest we can adapt the same method to run coverity here instead of fixing the code for test tool.